drivers/fastboot/fb_fsl/fb_fsl_partitions.c

```diff
@@ -46,14 +46,27 @@ enum {
 #endif
 	PTN_ALL_INDEX,
 	PTN_BOOTLOADER_INDEX,
+#ifdef CONFIG_FSL_FASTBOOT_BOOTLOADER_SECONDARY
+	PTN_BOOTLOADER_SECONDARY_INDEX,
+#endif
 #ifdef CONFIG_FSL_FASTBOOT_BOOTLOADER2
 	PTN_BOOTLOADER2_INDEX,
+	PTN_SIT_INDEX,
```

Comment:
Looks like `PTN_SIT_INDEX` won't get used unless `CONFIG_FSL_FASTBOOT_BOOTLOADER_SECONDARY` is defined.  Maybe move this under the #ifdef?

Reply:
right, will fix